core/scripts/cli/cli.js

```diff
@@ -1,11 +1,46 @@
-const cli = async function(callback) {
-  try {
-    console.log("You have started the UMA CLI!");
-  } catch (e) {
-    callback(e);
-  }
+const inquirer = require("inquirer");
+const vote = require("./vote");
+const wallet = require("./wallet");
 
-  callback();
+const collectInputs = async () => {
+  const prompts = [
+    {
+      type: "list",
+      name: "topMenu",
+      message: "Top level menu. What do you want to do?",
+      choices: ["wallet", "vote", "claim rewards", "DVM system status", "help", "exit"]
+    }
+  ];
+
+  return await inquirer.prompt(prompts);
 };
 
-module.exports = cli;
+async function run() {
+  let run = true;
+  while (run) {
+    const inputs = await collectInputs();
+    switch (inputs["topMenu"]) {
+      case "wallet":
+        await wallet(web3);
+        break;
+      case "vote":
+        await vote(web3, artifacts);
```

Comment:
`web3` and `artifacts` are available because we are in a `truffle exec` script right? Maybe we should explicitly retrieve these or set these just so they don't appear here for the first time

Reply:
we can do that. they are indeed here because of the truffle exec